preppal-be/routes/recipeApi.ts

```diff
@@ -90,10 +97,21 @@ routerRecipeApi.post("/createRecipe", async (req, res) => {
  */
 routerRecipeApi.post("/updateRecipe", async (req, res) => {
     try {
-        const { _id, author, title, description, image, ingredients, instructions, servingSize, prepTime, cookingTime, creationDate } = req.body;
+        const { _id, author, title, description, image, ingredients, instructions, servingSize, prepTime, cookingTime, creationDate, tags, visibility } = req.body;
+        
+        const verifyRecipe = await Recipe.findOne({ _id: _id });
+
+        if (!verifyRecipe) {
+            return res.status(400).json({ errors: [{ msg: "Invalid Id for recipe."}] });
+        } else if (!author) {
+            return res.status(404).json({ msg: "Recipe requires an author." });
```

Comment:
This should probably be a status 400 code
I think 404 means not found and 400 means bad request